source/Halibut.sln

```diff
@@ -122,58 +70,6 @@ Global
-		{A1E9A35D-F2CB-4223-8640-CC0E21E31C66}.Release|Mixed Platforms.Build.0 = Release|Any CPU
-		{A1E9A35D-F2CB-4223-8640-CC0E21E31C66}.Release|x86.ActiveCfg = Release|Any CPU
-		{549CD94C-DC6F-4C6A-B063-372E28726D38}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{549CD94C-DC6F-4C6A-B063-372E28726D38}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{549CD94C-DC6F-4C6A-B063-372E28726D38}.Debug|Mixed Platforms.ActiveCfg = Debug|Any CPU
-		{549CD94C-DC6F-4C6A-B063-372E28726D38}.Debug|Mixed Platforms.Build.0 = Debug|Any CPU
-		{549CD94C-DC6F-4C6A-B063-372E28726D38}.Debug|x86.ActiveCfg = Debug|Any CPU
-		{549CD94C-DC6F-4C6A-B063-372E28726D38}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{549CD94C-DC6F-4C6A-B063-372E28726D38}.Release|Any CPU.Build.0 = Release|Any CPU
-		{549CD94C-DC6F-4C6A-B063-372E28726D38}.Release|Mixed Platforms.ActiveCfg = Release|Any CPU
-		{549CD94C-DC6F-4C6A-B063-372E28726D38}.Release|Mixed Platforms.Build.0 = Release|Any CPU
-		{549CD94C-DC6F-4C6A-B063-372E28726D38}.Release|x86.ActiveCfg = Release|Any CPU
-		{4134E981-01AF-4398-8560-0F9BBA5480FB}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{4134E981-01AF-4398-8560-0F9BBA5480FB}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{4134E981-01AF-4398-8560-0F9BBA5480FB}.Debug|Mixed Platforms.ActiveCfg = Debug|Any CPU
-		{4134E981-01AF-4398-8560-0F9BBA5480FB}.Debug|Mixed Platforms.Build.0 = Debug|Any CPU
-		{4134E981-01AF-4398-8560-0F9BBA5480FB}.Debug|x86.ActiveCfg = Debug|Any CPU
-		{4134E981-01AF-4398-8560-0F9BBA5480FB}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{4134E981-01AF-4398-8560-0F9BBA5480FB}.Release|Any CPU.Build.0 = Release|Any CPU
-		{4134E981-01AF-4398-8560-0F9BBA5480FB}.Release|Mixed Platforms.ActiveCfg = Release|Any CPU
-		{4134E981-01AF-4398-8560-0F9BBA5480FB}.Release|Mixed Platforms.Build.0 = Release|Any CPU
-		{4134E981-01AF-4398-8560-0F9BBA5480FB}.Release|x86.ActiveCfg = Release|Any CPU
-		{0A152095-948F-4075-8FC9-D09B298C4460}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{0A152095-948F-4075-8FC9-D09B298C4460}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{0A152095-948F-4075-8FC9-D09B298C4460}.Debug|Mixed Platforms.ActiveCfg = Debug|Any CPU
-		{0A152095-948F-4075-8FC9-D09B298C4460}.Debug|Mixed Platforms.Build.0 = Debug|Any CPU
-		{0A152095-948F-4075-8FC9-D09B298C4460}.Debug|x86.ActiveCfg = Debug|Any CPU
-		{0A152095-948F-4075-8FC9-D09B298C4460}.Debug|x86.Build.0 = Debug|Any CPU
-		{0A152095-948F-4075-8FC9-D09B298C4460}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{0A152095-948F-4075-8FC9-D09B298C4460}.Release|Any CPU.Build.0 = Release|Any CPU
-		{0A152095-948F-4075-8FC9-D09B298C4460}.Release|Mixed Platforms.ActiveCfg = Release|Any CPU
-		{0A152095-948F-4075-8FC9-D09B298C4460}.Release|Mixed Platforms.Build.0 = Release|Any CPU
-		{0A152095-948F-4075-8FC9-D09B298C4460}.Release|x86.ActiveCfg = Release|Any CPU
-		{0A152095-948F-4075-8FC9-D09B298C4460}.Release|x86.Build.0 = Release|Any CPU
-		{55223D02-0CE9-4325-9A4D-5E88A544B633}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
The solution should not contain build configuration for the _build project otherwise nuke can't build the solution as it tries to build _build which is what nuke is running 😱